docs/guides/web-api-basics.md

```diff
@@ -266,8 +266,7 @@ CompletableFuture<ChatPostMessageResponse> postMessageResult = slack.methodsAsyn
 If your app has multiple nodes, it's also possible to specify the number of nodes.
 
 ```java
-import com.slack.api.methods.metrics.MetricsDatastore;
-import com.slack.api.methods.metrics.impl.MemoryMetricsDatastore;
+import com.slack.api.methods.metrics.MemoryMetricsDatastore;
```

Comment:
@WilliamBergamin or @seratch could you please confirm that it's safe to remove `MetricsDatastore` import?

Reply:
@mwbrooks this looks good to me 🚀